indexer_app/tasks.py

```
@@ -175,6 +323,121 @@ def update_account_statistics():
+                event.processed = process_vote_event(event_data, event.transaction_hash)
+            elif event_name == "c_pay":
+                event.processed = create_round_payout(event_data, event.transaction_hash)
+            elif event_name == "u_pay":
+
+                event.processed = update_round_payout(event_data, event.transaction_hash)
+            event.save()
+
+        except Exception as e:
+            jobs_logger.error(f"Error processing Stellar event { event_name, event.id}: {e}")
+
+    jobs_logger.info(f"Finished processing Stellar events.")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)